orion/pipelines/verified/dense_autoencoder/dense_autoencoder_realadexchange.json

```diff
@@ -0,0 +1,6 @@
+{
+    "mlprimitives.custom.timeseries_preprocessing.time_segments_aggregate#1": {
+        "time_column": "timestamp",
+        "interval": 600
```

Comment:
Sure, I was following branch [benchmark](https://github.com/signals-dev/Orion/tree/benchmark/orion/pipelines) when making this choice.

**Note** this will probably cause a change in the benchmarking results.